apps/exchange/lib/exchange/order_book.ex

```diff
@@ -367,10 +371,13 @@ defmodule Exchange.OrderBook do
         !Enum.empty?(Map.get(order_book.sell, pp))
       end)
       |> List.first()
-      new_ask_min = case new_ask_min do
+
+    new_ask_min =
+      case new_ask_min do
         nil -> 99_999
```

Comment:
I think  that now ask_min can't be nil ...  but we have to make sure...

Reply:
Yes it can and happens when the the sell side is empty